seed/lib/superperms/orgs/migrations/0022_auto_20220711_1928.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.13 on 2022-07-11 19:28
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('orgs', '0021_auto_20220622_2125'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
Looks like this migration could be combined with 0021. It's not necessary, just cleaner